src/Adapter/MSTest.CoreAdapter/TestMethodFilter.cs

```diff
@@ -95,5 +95,34 @@ internal object PropertyValueProvider(TestCase currentTest, string propertyName)
 
             return null;
         }
+
+        /// <summary>
+        /// Gets filter expression from run context.
+        /// </summary>
+        /// <param name="context">Run context</param>
+        /// <returns>Filter expression.</returns>
+        private ITestCaseFilterExpression GetTestCaseFilterFromRunContext(IRunContext context)
+        {
+            return context.GetTestCaseFilter(this.supportedProperties.Keys, this.PropertyProvider);
+        }
+
+        /// <summary>
+        /// Gets filter expression from discovery context.
+        /// </summary>
+        /// <param name="context">Discovery context</param>
+        /// <returns>Filter expression.</returns>
+        private ITestCaseFilterExpression GetTestCaseFilterFromDiscoveryContext(IDiscoveryContext context)
+        {
+            try
+            {
+                // GetTestCaseFilter is present in DiscoveryContext but not in IDiscoveryContext interface.
```

Comment:
Was this present in Dev11 as well? Did we validate this scenario in Dev11/Dev12?

Reply:
Validated in dev 12